libraries/Microsoft.Bot.Builder.Runtime/CoreBotAdapter.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using Microsoft.Bot.Builder;
+using Microsoft.Bot.Builder.Integration.AspNet.Core;
+using Microsoft.Bot.Builder.Runtime.Builders.Middleware;
+using Microsoft.Bot.Builder.Runtime.Settings;
+using Microsoft.Bot.Connector.Authentication;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+
+namespace Microsoft.Bot.Builder.Runtime
+{
+    public class CoreBotAdapter : BotFrameworkHttpAdapter
```

Comment:
@gabog Isn't the CloudAdapter rather limited right now?  Of course we'd want to match the current state of the SDK when that auth work is done.

Reply:
I think it is, you can maybe close this one, depending on timing, it may be good to align with that one that will eventually be the path going forward, I am not sure of how bug is the gap right now.

Reply:
I touch base with John and time it right.  Might be R13, but we'll get it in.